src/main/java/com/iexec/common/replicate/ReplicateStatusUpdate.java

```diff
@@ -59,13 +57,16 @@ public ReplicateStatusUpdate(ReplicateStatus status,
         this.modifier = modifier;
         this.details = details;
 
-        this.isSuccess = ReplicateStatus.isSuccess(status);
-
         if (modifier != null && modifier.equals(POOL_MANAGER)) {
            this.date = new Date();
         }
     }
 
+    @JsonIgnore
```

Comment:
Nice :)
Why not removing `@JsonIgnore` for @Pierre?

Reply:
I'm not sure to understand what you propose.
I had to add this annotation because for every method prefixed with `is` or `get`, there seem to be an assumption of an associated field.
If there is no such field, you need to tell it to the mapper through the `JsonIgnore` annotation.

Reply:
In order to satisfy Pierre's need, what about moving from `isSuccess()` to `success()`?

Reply:
I knew I didn't understand. I will try this and see if it allows to remove the annotation.

Reply:
Following this morning discussion and task feedback specifications, see cc3569e